src/cts/test/BUILD

```diff
@@ -0,0 +1,162 @@
+load("//test:regression.bzl", "regression_test")
+
+# From CMakeLists.txt or_integration_tests(TESTS
+COMPULSORY_TESTS = [
+    "array",
+    "array_ins_delay",
+    "array_no_blockages",
+    "balance_levels",
+    "check_buffers",
+    "check_buffers_blockages",
+    "check_buffers_blockages_merge",
+    "check_charBuf",
+    "check_max_fanout1",
+    "check_max_fanout2",
+    "check_wire_rc_cts",
+    "dummy_load",
+    "find_clock",
+    "find_clock_pad",
+    "insertion_delay",
+    "lvt_lib",
+    "max_cap",
+    "no_clocks",
+    "no_sinks",
+    "post_cts_opt",
+    "simple_test",
+    "simple_test_clustered",
+    "simple_test_clustered_max_cap",
+    "simple_test_hier",
+    "twice",
+]
+
+# Disabled in CMakeLists.txt
+MANUAL_TESTS = [
+    "cts_man_tcl_check",
+    "cts_readme_msgs_check",
+]
+
+ALL_TESTS = COMPULSORY_TESTS + MANUAL_TESTS
+
+filegroup(
+    name = "regression_resources",
+    # Dependencies could be specified more narrowly per test case,
+    # but at least it is not a glob of everything and there are
+    # per test glob patterns below.
+    srcs = [
+        "16sinks.def",
+        "Nangate45/Nangate45.lef",
+        "Nangate45/Nangate45.pdn.tcl",
+        "Nangate45/Nangate45.rc",
+        "Nangate45/Nangate45.rcx_rules",
+        "Nangate45/Nangate45.tracks",
+        "Nangate45/Nangate45.vars",
+        "Nangate45/Nangate45_fast.lib",
+        "Nangate45/Nangate45_lvt.lef",
+        "Nangate45/Nangate45_lvt.lib",
+        "Nangate45/Nangate45_slow.lib",
+        "Nangate45/Nangate45_stdcell.lef",
+        "Nangate45/Nangate45_tech.lef",
+        "Nangate45/Nangate45_typ.lib",
+        "Nangate45/fake_macros.lef",
+        "Nangate45/fake_macros.lib",
+        "Nangate45/fakeram45.cfg",
+        "Nangate45/fakeram45_1024x32.lef",
+        "Nangate45/fakeram45_1024x32.lib",
+        "Nangate45/fakeram45_256x16.lef",
+        "Nangate45/fakeram45_256x16.lib",
+        "Nangate45/fakeram45_512x64.lef",
+        "Nangate45/fakeram45_512x64.lib",
+        "Nangate45/fakeram45_64x32.lef",
+        "Nangate45/fakeram45_64x32.lib",
+        "Nangate45/fakeram45_64x7.lef",
+        "Nangate45/fakeram45_64x7.lib",
+        "Nangate45/fakeram45_64x96.lef",
+        "Nangate45/fakeram45_64x96.lib",
+        "Nangate45/work_around_yosys/cells.v",
+        "array_dummy.tcl",
+        "array_full_flow.tcl",
+        "array_tile.lef",
+        "array_tile.lib",
+        "array_tile_ins_delay.lef",
+        "array_tile_ins_delay.lib",
+        "check_lut.ok",
+        "create_lut.ok",
```

Comment:
Why are ok/defok/vok files in this list?

Reply:
I dont know what they are, I just make  best guess as a basis for further refinement.

Reply:
Ok, please remove *ok as those are the golden files and must be handled differently otherwise you would have to list them all.